Task 1 + 2 + 3

khaledkayyali opened the pull request:
removed JPA, used jdbc instead.
added Domains, DAO (DTO), Repository & Service.
added AppConfig & Example to test code.

Store is still in progress.

khaledkayyali committed:
Task 1 + 2 + 3
removed JPA, used jdbc instead.
added Domains, DAO (DTO), Repository & Service.
added AppConfig & Example to test code.

Store is still in progress.

Andriireva commented on src/main/java/com/sigma/sigmastore/dao/GoodsDao.java:
```diff
@@ -1,20 +1,16 @@
-
-    public GoodsDao( String name, double price, int units) {
-    }
+	String name;
+	double price;
+	int units;
```
make fields private

khaledkayyali replied:
Done

Andriireva commented on src/main/java/com/sigma/sigmastore/domain/Goods.java:
```diff
@@ -1,24 +1,16 @@
-        this.price = price;
-        this.units = units;
-    }
+	String name;
+	double price;
+	int units;
```
make fields private

khaledkayyali replied:
Done

Andriireva commented on src/main/java/com/sigma/sigmastore/domain/Store.java:
```diff
@@ -1,10 +1,14 @@
 
-    int Id;
-    String address;
+	int Id;
+	String address;
```
make fields private

khaledkayyali replied:
Done

Andriireva commented on src/main/java/com/sigma/sigmastore/repository/GoodsRepository.java:
```diff
@@ -11,54 +11,51 @@
-
-    }
+	private final JdbcTemplate jdbcTemplate;
+	private RowMapper<Goods> rowMapper;
```
since u make the upper field finale make this final as well. by the way making fields finals it is a best practice

khaledkayyali replied:
Done

khaledkayyali committed:
Task 1 + 2 + 3
removed JPA, used jdbc instead.
added Domains, DAO (DTO), Repository & Service.
added AppConfig & Example to test code.

Store is still in progress.

With changes you request
